### DISCUSSION TITLE
Enumerating residue interactions

### DISCUSSION THREAD

**Discussion by dhrblav** [molstar/molstar#541] [Q&A] [answered]
Hi,
Given a specific structure aa residue I would like to enumerate and investigate its interactions with surrounding elements.
How can this be achieved?
Thanks for your help.

**Comment by dsehnal** [comment, marked as answer]
Hi, have a look at for example ``https://github.com/molstar/molstar/blob/master/src/mol-plugin/behavior/dynamic/custom-props/computed/interactions.ts#L53`` which shows to to access interactions for a given ``Loci`` to create a hover label. You should be able to generalize that code.

**Reply by dhrblav** [reply]
Great, thank you!
